assets/samples/dynakube/v1beta5/kubernetesSecurityPostureManagement.yaml

```diff
@@ -0,0 +1,208 @@
+apiVersion: dynatrace.com/v1beta5
+kind: DynaKube
+metadata:
+  name: dynakube
+  namespace: dynatrace
+  # annotations:
+  #   feature.dynatrace.com/k8s-app-enabled: "true"
+spec:
+  # Link to api reference for further information: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters
+
+  # Required: Dynatrace apiUrl including the `/api` path at the end.
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Optional: Name of the secret holding the tokens used for connecting to Dynatrace.
+  #
+  # tokens: ""
+
+  # Optional: Defines a custom pull secret in case you use a private registry when pulling images from the Dynatrace environment
+  #
+  # customPullSecret: "custom-pull-secret"
+
+  # Optional: Disable certificate check for the connection between Dynatrace Operator and the Dynatrace Cluster.
+  #
+  # skipCertCheck: false
+
+  # Optional: Set custom proxy settings either directly or from a secret with the field 'proxy'
+  #
+  # proxy:
+  #   value: my-proxy-url.com
+  #   valueFrom: name-of-my-proxy-secret
+
+  # Optional: Add custom RootCAs from a configmap.
+  #
+  # trustedCAs: name-of-my-ca-configmap
+
+  # Optional: Set a network zone for the OneAgent and ActiveGate pods.
+  #
+  # networkZone: name-of-my-network-zone
+
+  # Optional: Configure istio to allow access to the Dynatrace Cluster from the OneAgent or ActiveGate pods.
+  #
+  # enableIstio: true
+
+  # Optional: Minimum minutes between Dynatrace API requests.
+  #
+  # dynatraceApiRequestThreshold: 15
+
+  # Configuration for Metadata Enrichment.
+  #
+  # metadataEnrichment:
+  #   enabled: true
+
+    # Optional: The namespaces where you want Dynatrace Operator to inject
+    #
+    # namespaceSelector:
+    #   matchLabels:
+    #     app: my-app
+    #   matchExpressions:
+    #   - key: app
+    #     operator: In
+  #     values: [my-frontend, my-backend, my-database]
+
+  kspm:
+    #  Define the host paths that are mounted to the container.
+    #
+    #  Default list of paths:
```

Comment:
"Default" is a little bit confusing. 

According to my ticket "By default no directories are mounted". This file is an example (so also kind of an instruction) so it should be clearly stated that this is rather a suggestion (smallest set of directories to get valuable knowledge about the host) or just a random example...

IMHO Default == kspm:{} -> all these directories are mounted
but it's not true.

Reply:
Thanks for the input. I agree, done: https://github.com/Dynatrace/dynatrace-operator/pull/4897/commits/4d37be28c3ff12b437b803a701fbe7f975176f79